cdap-ui/app/features/adapters/bottompanel.less

```diff
@@ -350,6 +350,10 @@ body.theme-cdap.state-adapters {
         font-weight: 500;
         font-size: 24px;
       }
+
+      h4.last-finished-time {
```

Comment:
Any specific reason why the finished time alone has different font-weight? Can we use ```<small></small>``` inside the ```h-tag```